components/src/main/java/uk/gov/hmrc/components/molecule/selectrow/SelectRowGroup.kt

```diff
@@ -143,9 +150,22 @@ class SelectRowGroup @JvmOverloads constructor(
             }
     }
 
+    private fun createDivider() = View(context).apply {
+        setBackgroundResource(R.drawable.components_divider)
+        layoutParams = LayoutParams(
+            ViewGroup.LayoutParams.MATCH_PARENT,
+            HMRC_SPACING_1.dpToPx().toInt()
+        ).apply {
+            marginStart = HMRC_SPACING_16.dpToPx().toInt()
+            marginEnd = HMRC_SPACING_16.dpToPx().toInt()
+        }
+    }
+
     companion object {
         private const val STATE_SELECTED_ROW = "STATE_SELECTED_ROW"
         private const val STATE_SUPER = "STATE_SUPER"
+        private const val HMRC_SPACING_16 = 16f
+        private const val HMRC_SPACING_1 = 1f
```

Comment:
Please can we stick to our already defined sizes?
`resources.getDimensionPixelSize(R.dimen.hmrc_spacing_16)`